### PR TITLE
Implement .say method

### PR DESCRIPTION
**Pull request by astj**
Defined as: https://docs.perl6.org/type/IO::Handle#method_say

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,11 @@ print(IO::Blob:D: *@text) returns Bool
 
 Text writing; writes the given `@text` to the Blob.
 
+say(IO::Blob:D: *@text) returns Bool
+------------------------------------
+
+Text writing; similar to write, but add a newline to end of the `@text`.
```
Comment (moznion) [inline README.md]: [Q] `similar to write` is right? Isn't it `similar to print`?

Comment (astj) [reply README.md]: Oops, you're right: https://github.com/moznion/p6-IO-Blob/pull/13/commits/7089988a60de844e10e106e49e60f323f9fd9ccd

Comment (moznion) [reply README.md]: :D

**Comment by moznion** [comment]
Thank you, great!

**Pull request merged**